Dockerfile

```diff
@@ -6,75 +6,109 @@ ARG NCPUS=1
 
 # R version 3.5.3 is not available via Conda, sticking to 3.5.1
 # ENV R_VERSION=${R_VERSION:-3.5.1}
-ENV R_VERSION=3.5.1
-ENV PY_VERSION=${PY_VERSION:-3.7}
-
-ENV USER=rstudio
-ENV LC_ALL=en_GB.UTF-8 \
-    LANG=en_GB.UTF-8
-
-ENV PATH /opt/conda/bin:$PATH
+ENV R_VERSION=3.5.1 \
+    PY_VERSION=${PY_VERSION:-3.7} \
+    USER=rstudio \
+    LC_ALL=en_GB.UTF-8 \
+    LANG=en_GB.UTF-8 \
+    PATH=/opt/conda/bin:$PATH
```

Comment:
I guess one benefit of having these set in separate `ENV` commands (and I guess these are relatively independent from each other) is that if we add an environment variable at the end of this list the previous layers would be cached.

But to be fair, `ENV` commands are pretty quick and these are also basically at the top of the `Dockerfile` so they would invalidate a lot of layers anyway 👌

Reply:
That was my exact thought process! We had a mix of methods so I just picked one and stuck with it.